src/io/json_reader.cpp

```diff
@@ -34,6 +32,11 @@ struct MyHandler {
 
     bool Double(double d) { w.setValue(key, d); return true; }
 
+    bool RawNumber(const char* str, rapidjson::SizeType len, bool copy)
+    {
+        w.setValue(key, str);
+        return true;
+    }
```

Comment:
What is this for? It seems unrelated to the 3rd party changes

Reply:
It is, see commit message of https://github.com/tue-robotics/ed/pull/144/commits/2a4bdb4149009bbca3e68ab5a1df720f498bd640